content/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-neovim.md

```diff
@@ -59,17 +63,23 @@ To use {% data variables.product.prodname_copilot %} in Neovim you must have Neo
 {% linux %}
 
 {% data reusables.copilot.install-copilot-in-neovim %}
-    - To install the {% data variables.product.prodname_copilot %} plugin directly, you must know where Neovim stores plugins. To install the plugin, enter the following command in Git Bash.
+   - To install {% data variables.product.prodname_copilot %} with a plugin manager, enter the following command in Terminal.
 
       ```
+      mkdir -p ~/.config/nvim/pack/github/start
       git clone https://github.com/github/copilot.vim \
-      ~/.config/PATH/TO/YOUR/NEOVIM/CONFIG/FILE/copilot.vim
+               ~/.config/nvim/pack/github/start/copilot.vim
       ```
+1. Visit https://github.com/settings/copilot and complete the required steps.
```

Comment:
```suggestion
```

Reply:
Hi @jules-p ,
The screenshot you sent 5 hours ago looks good!
I'm not able to see all of your changes together with the preview link (?),
looking at the diffs, I see two things,

1. This is an important step: Visit https://github.com/settings/copilot and complete the required steps.
1. The fence post (backticks) seem to have been removed? 

It is necessary to "select an option" in that link or copilot won't work with an unspecified error and the fence posts render code-blocks in the markdown.